dynawo/sources/Simulation/DYNSimulation.cpp

```diff
@@ -849,6 +849,12 @@ Simulation::checkCriteria(bool finalStep) {
   return criteriaChecked;
 }
 
+
+void
+Simulation::getFailingCriteria(std::vector<std::string>& failingCriteria) const {
+  data_->getFailingCriteria(failingCriteria);
+}
+
```

Comment:
As of today, this is not directly called in the dynawo flow. Is it done on purpose and will it be called by eventually an external code?

Reply:
Yes, this is done to collect the failing criteria directly from a simulation object in an external tool

Reply:
Fine, I just wanted to double check.